cmd/manifest/info.go

```diff
@@ -106,7 +106,7 @@ func getManifestInfo(ctx context.Context, clients *shared.ClientFactory, cmd *co
 	}
 	switch {
 	case source.Equals(config.MANIFEST_SOURCE_LOCAL):
-		return getManifestInfoProject(clients)
+		return getManifestInfoProject(ctx, clients)
 	case source.Equals(config.MANIFEST_SOURCE_REMOTE):
 		return getManifestInfoRemote(ctx, clients)
```

Comment:
note: It's nice to see that both `getManifestInfoProject(ctx, clients)` and `getManifestInfoRemote(ctx, clients)` have the same function signatures.